tests/test_html2docx.py

```diff
@@ -36,5 +36,5 @@ def test_html2docx(html_path, spec_path):
         for run, run_spec in zip(p.runs, runs_spec):
             assert run.text == run_spec["text"]
             for attr in ("bold", "italic", "underline", "subscript", "superscript"):
-                msg = f"Wrong {attr} for text '{run.text}' in {html_path}"
+                msg = f"Wrong {attr} for text '{run.text}' in {html_path.name}"
```

Comment:
I didn’t mind the full path, it avoids looking up where `p.html` is located. My preference would be the relative path from the git top level. OTOH, all test data is located under `test/data`, so the file can be expected to live there.

Reply:
I like the idea of using a relative path. I'll look into that.